machine/models/machine_dbs.py

```diff
@@ -0,0 +1,93 @@
+from odoo import models, fields, api, _
+
+from ..utils import generate_name_get
+
+
+class MachineDbsInstance(models.Model):
+    """Model representing database management system instance.
+
+    You can specify databases, users and other details on that instance.
+    """
+
+    _name = 'machine.dbs.instance'
+    _description = "Machine Instance Database System"
+
+    name = fields.Char(required=True,)
+    dbs_id = fields.Many2one('machine.dbs', "Database System", required=True)
+    machine_instance_id = fields.Many2one(
+        'machine.instance', "Machine Instance", required=True)
+    dbs_instance_user_ids = fields.One2many(
+        'machine.dbs.instance.user', 'dbs_instance_id', "DBS Users")
+    dbs_instance_db_ids = fields.One2many(
+        'machine.dbs.instance.database', 'dbs_instance_id', "Databases")
+    port = fields.Integer(required=True,)
+    # TODO: add possibility to specify users and databases.
+    amount_users = fields.Integer("Users Count")
+    amount_databases = fields.Integer("Databases Count")
```

Comment:
See TODO, it is left for some other time. If it would be done, then I would need to make it computed and specified manually (if for example you don't want to enter user by user, but just want to enter some amount).